static/scss/layout.scss

```diff
@@ -25,3 +26,14 @@
     bottom: 0;
 }
 
+// OmniPlayer styles
```

Comment:
At some point we should add a CSS linter, but in the mean time can you make the indentation consistent below?